messaging-experiments.json

```diff
@@ -265,8 +265,8 @@
   {
     "id": "PROTECTION-RETENTION-EXPERIMENT-RELEASE",
     "enabled": true,
-    "filter_expression": "env.channel == 'nightly' || env.channel == 'default'",
-    "targeting": "localeLanguageCode == 'en' && [userId, \"PROTECTION-RETENTION-EXPERIMENT-RELEASE\"]|stableSample(0.1)",
+    "filter_expression": "(env.channel == 'release' || env.channel == 'default') && env.version >= '77.'",
```

Comment:
Is the dot intended here in `77.`? To exclude `77`?

Just to confirm, do we have the required strings for those messages in 77?

Reply:
> https://github.com/mozilla/messaging-system-inflight-assets/blob/daf93389d7b4065871f70aa0591614634fca61af/messaging-experiments.yaml#L3

This is [Remote Settings syntax](https://searchfox.org/mozilla-central/rev/91d82d7cbf05a71954dfa49d0e43824c7c973e62/services/settings/remote-settings.js#70-73)
https://github.com/mozilla/messaging-system-inflight-assets/blob/daf93389d7b4065871f70aa0591614634fca61af/messaging-experiments.yaml#L3

> Just to confirm, do we have the required strings for those messages in 77?

Messages are `en` only and don't reference Fluent strings.